assignment/api/src/resources/assignment.properties

```diff
@@ -255,7 +255,7 @@ gen.thesubswill   = The submissions will be sent to {0} to be reviewed for plagi
 gen.thesubswill.indexed = The submissions will be sent to {0} to be reviewed for plagiarism. They will be indexed to be compared against by other submissions.
 gen.subStudentPreview = This assignment has enabled student preview which allows students to submit as many papers as a draft as they wish. A draft paper report is only visible to the owner of the paper, however instructors will be able to see the number of draft submissions and the report scores of each draft. Also, if there is a due date, the most recent draft will automatically be submitted if the student has no other submissions. Students can submit a draft paper as their final submission at any time within the plagiarism report.
 gen.onlythefoll   = <li>Only the following file types will be accepted: {0}.</li>
-gen.onlythefoll.turnitin = <li>File extension should always be included. Text after first point will be considered as file extension.</li><li>Attachments should not exceed 100 MB in size (.txt attachments should not exceed 2 MB).</li><li>Attachments should occupy less than 800 pages.</li><li>The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images.</li>
+gen.onlythefoll.turnitin = <li>The uploaded file must have a file extension, which is the text following the dot.</li><li>Attachments must be less than 100 MB in size (.txt attachments should be less than 2 MB), have a minimum of 20 words, and contain less than 800 pages.</li><li>The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images.</li>
```

Comment:
I don't think you need to say "which is the text following the dot"

Reply:
"The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images."

This is so obscure. How many people know what vector images are?

Reply:
@vacercon @josecebe I believe this comes from UPV, students from UPV uploading vector images and turnitin failing for this, am I right?

It's mentioned in the Turnitin support:
https://supportcenter.turnitin.com/s/article/M14-11-error

Reply:
Okay my opinion is that we should either be minimal and brief. Or we should list every single oddity/exception.

Reply:
I talked with @vacercon from UPV, and a proper solution would be to separate this vector images error to a list of possible technical errors.